notebooks/demo_individual/demo_convolution.ipynb

```diff
@@ -145,50 +145,80 @@
-      "\u001b[0;32m~/Desktop/osprey/venv/lib/python3.7/site-packages/requests/models.py\u001b[0m in \u001b[0;36mraise_for_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    939\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    940\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mhttp_error_msg\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 941\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mHTTPError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mhttp_error_msg\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mresponse\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    942\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    943\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mclose\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
-      "\u001b[0;31mHTTPError\u001b[0m: 500 Server Error: INTERNAL SERVER ERROR for url: http://localhost:5002/wps"
-     ]
+     "data": {
+      "text/plain": [
+       "convolutionResponse(\n",
+       "    output='http://localhost:5002/outputs/00644090-d6b2-11ea-a740-c86000e3f36f/sample.rvic.h0a.2013-01-01.nc'\n",
+       ")"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
+   "source": [
+    "output.get()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
    "source": [
     "# generate convolution\n",
     "output = osprey.convolution(\n",
-    "    config = cfg_file\n",
+    "    config = '''{\n",
+    "        \"OPTIONS\":{\n",
+    "            \"CASEID\":\"sample\", \n",
+    "            \"RUN_STARTDATE\": \"2012-12-01-00\", \n",
+    "            \"STOP_DATE\": \"2012-12-31\"\n",
+    "        },\n",
+    "        \"DOMAIN\":{\n",
+    "            \"FILE_NAME\":\"https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/sample_routing_domain.nc\"\n",
+    "        },\n",
+    "        \"PARAM_FILE\":{\n",
+    "            \"FILE_NAME\":\"https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/sample.rvic.prm.COLUMBIA.20180516.nc\"\n",
+    "        },\n",
+    "        \"INPUT_FORCINGS\": {\n",
+    "            \"DATL_PATH\": \"https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/\",\n",
+    "            \"DATL_FILE\": \"columbia_vicset2.nc\"\n",
+    "        }\n",
+    "    }'''\n",
```

Comment:
So the code is supposed to allow us to input a `dict` instead of a config file. In this case we are not passing in a `dict` but a `str`. I tried changing this code to an actual `dict` and the code didn't work. I know that the bird will translate this param to a `str` anyways, but we can worry about running any translation in the bird rather than here in `jupyter`.